DashAI/back/core/config.py

```diff
@@ -36,6 +36,7 @@ class Settings(BaseSettings):
     DB_PATH: str = "DashAI/back/database/DashAI.sqlite"
     FRONT_BUILD_PATH: str = "DashAI/front/build"
     USER_DATASET_PATH: str = "DashAI/back/user_datasets"
+    USER_RUN_PATH: str = "DashAI/back/user_runs"
```

Comment:
Maybe to discuss in the future, but wouldn't it be nice to save the user's artifacts in the user's home? The user will probably never find the path where the package is installed.